src/App.js

```diff
@@ -9,17 +9,15 @@ import DisplayEntries from './components/DisplayEntries';
 import SumFields from './components/SumField';
 
 
-
-
 class App extends Component {
   render() {
     return (
       <Provider store={store}>
-        <Col className="App" horizontalAlignment={'Center'}>
+        <Col className="App" horizontalAlignment={HorizontalAlignment.CENTER}>
             
             <Text 
               fontStyle={0}
```

Comment:
`FontStyle.TITLE` would be the preferred way to use this, but it's a bit weird to use the component library without TS as we haven't specified `proptypes` for any component :)

Reply:
Whoops, missed one... 
TS is comming!